programs/nft-voter/src/instructions/configure_collection.rs

```diff
@@ -0,0 +1,104 @@
+use anchor_lang::{
+    account,
+    prelude::{Context, Signer},
+    Accounts,
+};
+
+use anchor_lang::prelude::*;
+use anchor_spl::token::Mint;
+use spl_governance::state::realm;
+
+use crate::state::{CollectionConfig, Registrar};
+use crate::{error::NftVoterError, state::MaxVoterWeightRecord};
+
+/// Configures NFT voting collection which defines what NFTs can be used for governances
+/// and what weight they have
+/// The instruction updates MaxVoterWeightRecord which is used by spl-gov to determine max voting power
+/// used to calculate voting quorum    
+#[derive(Accounts)]
+pub struct ConfigureCollection<'info> {
+    /// Registrar for which we configure this Collection
+    #[account(mut,
+        constraint = registrar.realm == realm.key() 
```

Comment:
nitpick: you can also replace this with a simpler `has_one = realm` check

Reply:
or `address = registrar.realm`

Reply:
Super cool, thank you.  The direction of the constraint `address = registrar.realm` is much better